### PR TITLE
[feature] Add configuration files

### PR DESCRIPTION
**Pull request by YuanmingLeee**
1. Model weight configuration
2. Device placement configuration

### CODE REVIEW

**Review by HuaizhengZhang — APPROVED** [review]

**Comment by HuaizhengZhang** [comment]
LGTM.

Have these been tested?

**Comment by YuanmingLeee** [comment]
> LGTM.
> 
> Have these been tested?

Yes

File: server/config/weight_config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  places365:
+    backbone: resnet50
+    model: places365/{}.pth
+    label: places365/categories.txt
+  sound_net: soundnet/soundnet_fr.pb
+feature_extraction:
+  sentence_encoder: sentence
+  vggish:
+    fr: audioset/vggish_fr.pb
+    pca: vggish_pca_params.npz
```
Comment (QimingAi) [inline server/config/weight_config.yml:24]:
```
pca: audioset/vggish_pca_params.npz
```

Comment (QimingAi) [reply server/config/weight_config.yml:24]: fixed in PR #35

**Pull request merged**